client/src/app/modules/events/participant-list/invite-dialog/invite-dialog.component.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { translateLoaderSpy } from './../../../../utils-test/test-utils.spec';
+import { NgbModal, NgbModule, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { HttpClient } from '@angular/common/http';
+import { TranslateModule, TranslateLoader, TranslateService } from '@ngx-translate/core';
+import { TagInputModule } from 'ngx-chips';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { InviteDialogComponent } from './invite-dialog.component';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpLoaderFactory } from '../../../../app.module';
+import { IEventSchema } from '../../../../../../../server/src/interfaces/mongo.interface';
```

Comment:
Not sure if this is the right way how to do it. We are handling 2 projects in 1 respository. And we can have different model on client side (as we decide not to send other properties of the object).

I'd prefer to create a model in client as well.

Reply:
Autoimport "typo". Thanks for noticing.